README.md

```diff
@@ -676,12 +676,12 @@ class { 'grafana':
     apiVersion  => 1,
     datasources => [
       {
-	    name      => 'Prometheus',
-		type      => 'prometheus',
-		access    => 'proxy',
-		url       => 'http://localhost:9090/prometheus',
-		isDefault => true,
-	  },
+      name      => 'Prometheus',
```

Comment:
Can you recheck the alignment here? I think it was correct before and is now broken.

Reply:
I reviewed the entire README and it looks like I misaligned a few things after switching to soft tabs. I think it is fixed now.